docs/docs/en/guide/project/workflow-definition.md

```diff
@@ -50,6 +50,17 @@ Click the `Save` button, and the "Set DAG chart name" window pops up, as shown i
 
 ![workflow-save](../../../../img/new_ui/dev/project/workflow-save.png)
 
+### Configure workflow (process) execution type
+
+Click the `Save` button and configure `process execution type` in the pop-up window. There are four process execution types:
+
+- `Parallel`: If there are multiple instances of the same workflow definition, execute the instances in parallel.
+- `Serial Wait`: If there are multiple instances of the same workflow definition, execute the instances in serial.
+- `Serial Discard`: If there are multiple instances of the same workflow definition, discard the later ones.
```

Comment:
I'm not sure whether I got it correct here. Could u plz help take a look when available? @ruanwenjun @caishunfeng Thanks~

Reply:
`Serial Discard`: If there are multiple instances of the same workflow definition, discard the later ones and kill the current running ones.

Reply:
> `Serial Discard`: If there are multiple instances of the same workflow definition, discard the later ones and kill the current running ones.

Just for confirmation, in that case, there will be no instances running for the workflow definition once killing done?

Reply:
> Just for confirmation, in that case, there will be no instances running for the workflow definition once killing done?

Yes, if a new workflow instance ready to run, it will kill all the running instances if serial discard.
see https://github.com/apache/dolphinscheduler/blob/82ddd72e4a2fae8d8926e5ff8ec35600d0483fbc/dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessServiceImpl.java#L343-L356

Reply:
@caishunfeng Got it, thanks for the explanation. BTW, I've already updated the docs. PTAL.